core/src/main/java/com/github/kaktushose/jda/commands/embeds/error/DefaultErrorMessageFactory.java

```diff
@@ -52,13 +59,23 @@ public MessageCreateData getTypeAdaptingFailedMessage(@NotNull ErrorContext cont
                         .map(OptionDataDefinition::name)
                         .collect(Collectors.joining(" ")));
                 expected = commandOption.declaredType().getSimpleName();
-                actual = humanReadableType(optionMapping);
+                actual = Helpers.humanReadableType(optionMapping);
                 input = optionMapping.getAsString();
                 break;
             } else {
                 name = "%s %s".formatted(name, commandOption.name());
             }
         }
+
+        if (embeds.exists("typeAdaptingFailed")) {
```

Comment:
The possibility to replace the default embeds should be documented somehow. This also applies for the other instances of this functionality below.